app/src/main/java/net/aiscope/gdd_app/extensions/ViewExt.kt

```diff
@@ -0,0 +1,25 @@
+package net.aiscope.gdd_app.extensions
+
+import android.view.View
+
+fun View.isVisible() = this.visibility == View.VISIBLE
+
+fun View.visible() {
+    this.visibility = View.VISIBLE
+}
+
+fun View.invisible() {
+    this.visibility = View.INVISIBLE
+}
+
+fun View.gone() {
+    this.visibility = View.GONE
+}
+
+fun View.selectVisibility(shouldBeVisible: Boolean) {
+    this.visibility = if (shouldBeVisible) View.VISIBLE else View.GONE
+}
+
+fun View.selectInvisibility(shouldBeInvisible: Boolean) {
+    this.visibility = if (shouldBeInvisible) View.INVISIBLE else View.VISIBLE
+}
```

Comment:
Hello, no problem, I will do it tomorrow.

Just to be sure, you mean not using the extension at all or just deleting  the isVisible() extension? Because it's the only one I see in core-ktx.

Reply:
Besides `isVisible`, there's also `isGone` and `isInvisible`, you can pick and choose which works best on a use-case basis. And they all work both as getters and setters.

Reply:
Done.  Really good catch, I wasn't aware that they added this in core ktx :)